### PR TITLE
Measurement fMRI + Test

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Comment by egolol** [comment]
Still a draft

**Comment by giovannivolpe** [comment]
> Still a draft

Ok, I removed myself as a reviewer and I'll wait for you to re-assign me before revising it

**Comment by egolol** [comment]
ready to review @annacanal

File: braph2/workflows/fMRI/MeasurementfMRI.m [hunk 1]
```diff
@@ -0,0 +1,121 @@
+classdef MeasurementfMRI < Measurement
+    % single group of fMRI subjects
+    properties
+        values  % array with the values of the measure for each subject
+        average_value  % average value of the group
+    end
+    methods  % Constructor
+        function m =  MeasurementfMRI(id, label, notes, atlas, measure_code, group, varargin)
+            
+            graph_type = AnalysisMRI.getGraphType();
```
Comment (annacanal) [inline braph2/workflows/fMRI/MeasurementfMRI.m:10]: Analysis**f**MRI

Comment (egolol) [reply braph2/workflows/fMRI/MeasurementfMRI.m:10]: yikes! ty

**Pull request merged**